run-bungeecord.sh

```diff
@@ -52,14 +52,32 @@ if [ -d /config ]; then
     cp -u /config/config.yml "$BUNGEE_HOME/config.yml"
 fi
 
-if [ -f /var/run/default-config.yml -a ! -f /server/config.yml ]; then
+if [ -f /var/run/default-config.yml -a ! -f $BUNGEE_HOME/config.yml ]; then
     echo "Installing default configuration"
-    cp /var/run/default-config.yml /server/config.yml
+    cp /var/run/default-config.yml /config.yml
```

Comment:
I think it should be
```suggestion
    cp /var/run/default-config.yml $BUNGEE_HOME/config.yml
```

Reply:
Yes its an misstake